filters/ignore_new_executors.hpp

```diff
@@ -0,0 +1,56 @@
+#include <ctime>
+#include <memory>
+
+#include "messages/serenity.hpp"
+
+#include "serenity/executor_map.hpp"
+#include "serenity/serenity.hpp"
+
+#include "stout/option.hpp"
+
+#ifndef SERENITY_IGNORE_NEW_TASKS_HPP
```

Comment:
In my filters i use FILTER suffix - then it would looks like "SERENITY_IGNORE_NEW_TASKS_FILTER_HPP". IMO it should as unique as possible.. (: